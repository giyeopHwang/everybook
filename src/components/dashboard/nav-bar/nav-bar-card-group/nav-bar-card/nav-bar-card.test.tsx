import { renderWithWrapper } from '@/test/test-utils';
import NavBarCard from './nav-bar-card.component';
import { mockNet, mockPlus } from '@/test/test-mock';

describe('NavBarCard', () => {
  describe('without details prop', () => {
    it('renders NavBarCard component', () => {
      const { title, amount, type } = mockNet;
      const navBarCard = renderWithWrapper(
        <NavBarCard title={title} amount={amount} type={type} />
      );

      expect(navBarCard).toMatchSnapshot();
    });

    it('renders collapsed when specified', () => {
      const { title, amount, type } = mockNet;
      const navBarCard = renderWithWrapper(
        <NavBarCard
          title={title}
          amount={amount}
          type={type}
          expanded={false}
        />
      );

      expect(navBarCard).toMatchSnapshot();
    });
  });

  describe('with details prop', () => {
    it('renders NavBarCard component', () => {
      const { title, amount, type, details } = mockPlus;
      const navBarCard = renderWithWrapper(
        <NavBarCard
          title={title}
          amount={amount}
          type={type}
          details={details}
        />
      );

      expect(navBarCard).toMatchSnapshot();
    });

    it('renders collapsed when specified', () => {
      const { title, amount, type, details } = mockPlus;
      const navBarCard = renderWithWrapper(
        <NavBarCard
          title={title}
          amount={amount}
          type={type}
          details={details}
          expanded={false}
        />
      );

      expect(navBarCard).toMatchSnapshot();
    });
  });
});
