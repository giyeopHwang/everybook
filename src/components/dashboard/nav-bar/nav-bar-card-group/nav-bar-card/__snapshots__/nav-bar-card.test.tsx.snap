// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavBarCard > with details prop > renders NavBarCard component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-c1a362"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="AddIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              PLUS
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            300
          </p>
        </div>
        <section
          class="css-1r2jysi MuiBox-root css-0"
        >
          <div
            class="css-uf1ume MuiBox-root css-0"
          >
            <div
              class="css-44nm6q MuiBox-root css-0"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-r6guz9-MuiSvgIcon-root"
                data-testid="SubdirectoryArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m19 15-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z"
                />
              </svg>
              <span
                class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
              >
                PLUS_ITEM_1
              </span>
            </div>
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              100
            </span>
          </div>
          <div
            class="css-uf1ume MuiBox-root css-0"
          >
            <div
              class="css-44nm6q MuiBox-root css-0"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-r6guz9-MuiSvgIcon-root"
                data-testid="SubdirectoryArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m19 15-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z"
                />
              </svg>
              <span
                class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
              >
                PLUS_ITEM_2
              </span>
            </div>
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              200
            </span>
          </div>
        </section>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-c1a362"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="AddIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            PLUS
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          300
        </p>
      </div>
      <section
        class="css-1r2jysi MuiBox-root css-0"
      >
        <div
          class="css-uf1ume MuiBox-root css-0"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-r6guz9-MuiSvgIcon-root"
              data-testid="SubdirectoryArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m19 15-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z"
              />
            </svg>
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              PLUS_ITEM_1
            </span>
          </div>
          <span
            class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
          >
            100
          </span>
        </div>
        <div
          class="css-uf1ume MuiBox-root css-0"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-r6guz9-MuiSvgIcon-root"
              data-testid="SubdirectoryArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m19 15-6 6-1.42-1.42L15.17 16H4V4h2v10h9.17l-3.59-3.58L13 9l6 6z"
              />
            </svg>
            <span
              class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
            >
              PLUS_ITEM_2
            </span>
          </div>
          <span
            class="MuiTypography-root MuiTypography-caption css-1sn4lm3-MuiTypography-root"
          >
            200
          </span>
        </div>
      </section>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;

exports[`NavBarCard > with details prop > renders collapsed when specified 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-c1a362"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="AddIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              PLUS
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            300
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-c1a362"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="AddIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            PLUS
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          300
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;

exports[`NavBarCard > with details prop > renders with hidden amount 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-c1a362"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="AddIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              PLUS
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            *********
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-c1a362"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="AddIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            PLUS
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          *********
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;

exports[`NavBarCard > without details prop > renders NavBarCard component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-16qv2i2"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="TollIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              NET
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            100
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-16qv2i2"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="TollIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            NET
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          100
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;

exports[`NavBarCard > without details prop > renders collapsed when specified 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-16qv2i2"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="TollIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              NET
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            100
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-16qv2i2"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="TollIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            NET
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          100
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;

exports[`NavBarCard > without details prop > renders with hidden amount 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-1xqftfh MuiBox-root css-12y2hfg"
      >
        <div
          class="css-pe6vce MuiBox-root css-16qv2i2"
        >
          <div
            class="css-44nm6q MuiBox-root css-0"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
              data-testid="TollIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
              />
            </svg>
            <p
              class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
            >
              NET
            </p>
          </div>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            *********
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-1xqftfh MuiBox-root css-12y2hfg"
    >
      <div
        class="css-pe6vce MuiBox-root css-16qv2i2"
      >
        <div
          class="css-44nm6q MuiBox-root css-0"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1u2zctn-MuiSvgIcon-root"
            data-testid="TollIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M15 4c-4.42 0-8 3.58-8 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zM3 12c0-2.61 1.67-4.83 4-5.65V4.26C3.55 5.15 1 8.27 1 12s2.55 6.85 6 7.74v-2.09c-2.33-.82-4-3.04-4-5.65z"
            />
          </svg>
          <p
            class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
          >
            NET
          </p>
        </div>
        <p
          class="MuiTypography-root MuiTypography-body2 css-1d3njat-MuiTypography-root"
        >
          *********
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
